src/System.IO.Pipelines.Networking.Tls/project.json

```diff
@@ -0,0 +1,31 @@
+﻿{
+  "version": "0.1.0-*",
+  "description": "Secure Pipelines implementation using TLS that beats the pants off System.Net.Security.SslStream",
```

Comment:
agreed, it was a hang over from less "official" days changed (surprised you were the first to notice) changed